framework/platform/bpl/uci/cfg/bpl_cfg_uci.h

```diff
@@ -68,7 +68,14 @@ enum paramType { TYPE_RADIO = 0, TYPE_VAP };
 
 #include <slibc/stdio.h>
 #include <slibc/string.h>
+
+extern "C" {
```

Comment:
Is this used by any platform except for RDKB? If not, why didn't you wrap it also with #ifdef?

Reply:
It is wrapped with #ifdef